src/vs/server/node/webClientServer.ts

```diff
@@ -100,6 +107,10 @@ export class WebClientServer {
 				// callback support
 				return this._handleCallback(res);
 			}
+			if (/^\/extensionResource\//.test(pathname)) {
```

Comment:
Should we rename this to `web-extension-resource`? Lowercase to align with typical paths used in HTTP requests, and prepending `web-` to clarify that this is only about web worker extensions i.e. not about extensions installed on the server, which might also have resources.

Reply:
Done